bitnami/kafka/values.yaml

```diff
@@ -198,6 +198,9 @@ socketSendBufferBytes: 102400
 ## @param zookeeperConnectionTimeoutMs Timeout in ms for connecting to Zookeeper
 ##
 zookeeperConnectionTimeoutMs: 6000
+## @param zookeeperChrootPath Path which puts data under some path in the global ZooKeeper namespace
+##
+zookeeperChrootPath: ""
```

Comment:
Could you improve the paramter description? Why is it called `chroot`?

Reply:
This term and description from kafka documentation: https://kafka.apache.org/documentation/#brokerconfigs_zookeeper.connect

> The server can also have a ZooKeeper chroot path as part of its ZooKeeper connection string which puts its data under some path in the global ZooKeeper namespace. For example to give a chroot path of `/chroot/path` you would give the connection string as `hostname1:port1,hostname2:port2,hostname3:port3/chroot/path`.

I can add an example how it's in the official doc.

Reply:
I thinks it's enough if we add this references in the metadata:
```suggestion
## @param zookeeperChrootPath Path which puts data under some path in the global ZooKeeper namespace
## ref: https://kafka.apache.org/documentation/#brokerconfigs_zookeeper.connect
##
zookeeperChrootPath: ""
```